vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/EmailField.java

```diff
@@ -24,6 +24,7 @@
 import com.vaadin.flow.component.HasValidation;
 import com.vaadin.flow.component.InputNotifier;
 import com.vaadin.flow.component.KeyNotifier;
+import com.vaadin.flow.component.HasClearButton;
```

Comment:
nit: let's place this import before the `com.vaadin.flow.component.HasHelper` import.